casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -250,6 +250,7 @@ class RuntimeManager private (val emptyStateHash: ByteString, runtimeContainer:
                              case Right(newCheckpoint) =>
                                doReplayEval(rem, newCheckpoint.root)
                              case Left(ex: ReplayException) =>
+                               throw new Error("Fail")
```

Comment:
WARN:
1. The exception `ex` is ignored. Even not logged. Consider adding it as an argument to `Error`.
2. Very generic exception is thrown `Error` without any context information. This will make troubleshooting rather painful. Consider `Task.fail`.